### ISSUE TITLE
Pod env variables from a config map not showing up on pod page

### ISSUE THREAD

**Issue by spadgett** [openshift/origin-web-console#862]
@jwforres @benjaminapetersen 

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/20222438/54d91a7e-a803-11e6-9ef7-e0ef47cdef0b.png)

```yaml
      env:
        - name: SPECIAL_LEVEL_KEY
          valueFrom:
            configMapKeyRef:
              name: special-config
              key: special.how
        - name: SPECIAL_TYPE_KEY
          valueFrom:
            configMapKeyRef:
              name: special-config
              key: special.type
```

**Comment by jwforres** [comment]
@spadgett this is only happening on the pod page?

**Comment by benjaminapetersen** [comment]
@jwforres yeah, think we got all the other ones in the big kve update.  I just missed the `$filter` for this one :frowning:

**Comment by jwforres** [comment]
@benjaminapetersen if the fix is simple i'd consider this for 1.4
